lib/MetaCPAN/Model/Search.pm

```diff
@@ -222,8 +222,14 @@ sub build_query {
     $params //= {};
     ( my $clean = $search_term ) =~ s/::/ /g;
 
-    my $negative
-        = { term => { 'mime' => { value => 'text/x-script.perl' } } };
+    my $negative = {
+        bool => {
+            should => [
+                { term => { 'mime' => { value => 'text/x-script.perl' } } },
+                { term => { 'deprecated' => { value => 1, boost => -100 } } },
```

Comment:
What is the value 100 based on? for positive matches we boost by 20. 
I'm OK with it, but we do need to see what the effect of it will be and tweak it later if needed.

Reply:
Honestly, "just a number that I pulled out of thin air".

And yes... **totally** agree that it will be something that we would need to see the effect of on a larger index, and then tweak/adjust as necessary.

Reply:
Ok. Let's see what happens. :)